Core/include/AnalysisCore/TrackSelectorPID.h

```diff
@@ -471,7 +471,7 @@ class TrackSelectorPID
         isSelRICH = false; // is selected as pion by RICH
       }
     } else {
-      isSelRICH = false;
+      isSelRICH = true;
     }
```

Comment:
This does not seem right.

Reply:
> This does not seem right.
If the tracks p > 0.6(not use tod) and no rich response, It will be rejected if isSelRICH = false; I think this is not what we expected.

Reply:
The function is supposed to tell
> whether a track is identified as electron and rejected as pion by TOF or RICH.

That means that a particle has to pass the electron selection and fail the pion selection in a given detector.
This cannot be satisfied if the detector response is not available or disabled. In that case the track is not selected as electron and therefore the results should be `false`.

What is the behaviour that you would expect?